src/pspm_init.m

```diff
@@ -989,22 +989,22 @@
     'default', 0);
 
 defaults.glm(9) = ... % GLM for SEBR (fear-conditioning)
-    struct('modality', 'emg_pp', ...                               % modality name
-    'modelspec', 'sebr', ...                                  % modality name
-    'cbf', struct('fhandle', @pspm_bf_sebrf, 'args', 0), ...  % default basis function/set
-    'filter', struct('lpfreq', NaN, 'lporder', NaN,  ...        % default filter settings
+    struct('modality', 'emg_pp', ...
+    'modelspec', 'sebr', ...
+    'cbf', struct('fhandle', @pspm_bf_sebrf, 'args', 0), ...
+    'filter', struct('lpfreq', NaN, 'lporder', NaN,  ...
     'hpfreq', NaN, 'hporder', NaN, 'down', 1000, ...
     'direction', 'uni'), ...
     'default', 1);
 
 defaults.glm(10) = ... % GLM for Scanpath-speed
-    struct('modality', 'sps', ...                               % modality name
-    'modelspec', 'sps', ...                                  % modality name
-    'cbf', struct('fhandle', @pspm_bf_spsrf_box, 'args', 1), ...  % default basis function/set
-    'filter', struct('lpfreq', NaN, 'lporder', NaN,  ...        % default filter settings
+    struct('modality', 'sps', ...
+    'modelspec', 'sps', ...
+    'cbf', struct('fhandle', @pspm_bf_spsrf_box, 'args', 1), ...

```

Comment:
Hi, thank you for the explaining. I think I have got more understanding of the issue in this model. I think both `pspm_init` and `pspm_bf_spsrf_box` (or together with other potential similar functions) shall be updated. However, I am not sure if this feature is included in the test function correctly. May I know if you mind sending me one or two lines of code showing how this model is used by `pspm_bf_spsrf_box`? This would make it easier for me to update the test function as well. Many thanks!